### PR TITLE
Internal: Fix document test for WP nightly

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by yotamselementor — APPROVED** [review]
👏

File: tests/playwright/sanity/core/base/document.test.ts [hunk 1]
```diff
@@ -51,7 +51,13 @@ test.describe( 'Document tests', async () => {
 
 async function addElement( wpAdmin: WpAdminPage, elementType: string ) {
 	const frame = wpAdmin.page.frame( { name: 'editor-canvas' } );
-	await frame.click( '.block-editor-inserter__toggle' );
+	if ( ! await wpAdmin.page.frameLocator( 'iframe[name="editor-canvas"]' ).locator( 'p[role="document"]' ).isVisible() ) {
```
Comment (yotamselementor) [inline tests/playwright/sanity/core/base/document.test.ts:54]: How about doing something like this once:
```typescript
await wait wpAdmin.page.frameLocator( 'iframe[name="editor-canvas"]' ).addLocatorHandler('p[role="document"]', async overlay => {
  await frame.locator( '.block-editor-inserter__toggle' ).click();
}, { times: 3, noWaitAfter: true });
```

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/core/base/document.test.ts (hunk 1 above)
Comment (yotamselementor) [reply tests/playwright/sanity/core/base/document.test.ts:54]: To be clear - this needs to be done after page load, just once, not here

Comment (yotamselementor) [reply tests/playwright/sanity/core/base/document.test.ts:54]: see: https://playwright.dev/docs/api/class-page#page-add-locator-handler

Comment (dmaziar) [reply tests/playwright/sanity/core/base/document.test.ts:54]: I'm not sure I understand the logic. You want to click + button 3 times?
The problem here is, that first time Playwright clicks + button, it is in iframe. Next + button is not.

Comment (dmaziar) [reply tests/playwright/sanity/core/base/document.test.ts:54]: > see: https://playwright.dev/docs/api/class-page#page-add-locator-handler

I'm not sure this is a case for that. Here we have a clear case. If no element was added- + in Iframe, if at least one is there, then it is not in iframe

Comment (yotamselementor) [reply tests/playwright/sanity/core/base/document.test.ts:54]: I want that after the iframe is rendered to add a locator handler that will make these popups go away, regardless of what action it is we perform

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**